balena/models/organization.py

```diff
@@ -161,3 +162,143 @@ def remove(self, org_id):
             'organization({org_id})'.format(org_id=org_id), 'DELETE',
             endpoint=self.settings.get('pine_endpoint'), login=True
         )
+
+
+class OrganizationInvite():
+    """
+    This class implements organization invite model for balena python SDK.
+
+    """
+
+    def __init__(self):
+        self.base_request = BaseRequest()
+        self.settings = Settings()
+        self.config = Config()
+        self.auth = Auth()
+        self.RESOURCE = 'invitee__is_invited_to__organization'
+
+    def get_all(self):
+        """
+        Get all invites.
+
+        Returns:
+            list: list contains info of invites.
+            
+        Examples:
+            >>> balena.models.organization.invite.get_all()
+            [{'id': 2862, 'message': 'Test invite', 'invitee': {'__id': 2965, '__deferred': {'uri': '/resin/invitee(@id)?@id=2965'}}, 'is_invited_to__organization': {'__id': 26474, '__deferred': {'uri': '/resin/organization(@id)?@id=26474'}}, 'is_created_by__user': {'__id': 5227, '__deferred': {'uri': '/resin/user(@id)?@id=5227'}}, 'organization_membership_role': {'__id': 2, '__deferred': {'uri': '/resin/organization_membership_role(@id)?@id=2'}}, '__metadata': {'uri': '/resin/invitee__is_invited_to__organization(@id)?@id=2862'}}]
+
+        """
+
+        return self.base_request.request(
+            self.RESOURCE, 'GET', endpoint=self.settings.get('pine_endpoint')
+        )['d']
+        
+    def get_all_by_organization(self, org_id):
+        """
+        Get all invites by organization.
+
+        Args:
+            org_id (str): organization id.
+
+        Returns:
+            list: list contains info of invites.
+            
+        Examples:
+            >>> balena.models.organization.invite.get_all_by_organization(26474)
+            [{'id': 2862, 'message': 'Test invite', 'invitee': {'__id': 2965, '__deferred': {'uri': '/resin/invitee(@id)?@id=2965'}}, 'is_invited_to__organization': {'__id': 26474, '__deferred': {'uri': '/resin/organization(@id)?@id=26474'}}, 'is_created_by__user': {'__id': 5227, '__deferred': {'uri': '/resin/user(@id)?@id=5227'}}, 'organization_membership_role': {'__id': 2, '__deferred': {'uri': '/resin/organization_membership_role(@id)?@id=2'}}, '__metadata': {'uri': '/resin/invitee__is_invited_to__organization(@id)?@id=2862'}}]
+
+        """
+
+        params = {
+            'filter': 'is_invited_to__organization',
+            'eq': org_id
+        }
+
+        return self.base_request.request(
+            self.RESOURCE, 'GET', params=params,
+            endpoint=self.settings.get('pine_endpoint')
+        )['d']
+        
+    def create(self, org_id, invitee, role_name=None, message=None):
+        """
+        Creates a new invite for an organization.
+
+        Args:
+            org_id (str): organization id.
+            invitee (str): the email/balena_username of the invitee.
```

Comment:
That's actually something we need to change in the JS SDK as well
```suggestion
            invitee (str): the email of the invitee.
```
See: https://github.com/balena-io/balena-api/blob/a0472380577b2fd516c3d64929fb42af86830b22/src/features/invitations/hooks/application.ts#L74-L76